applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.cpp

```diff
@@ -0,0 +1,4943 @@
+void SymbolicStokes<TElementData>::CalculateRightHandSide(
+    VectorType &rRightHandSideVector,
+    ProcessInfo &rCurrentProcessInfo)
+{
+    MatrixType tmp;
+    CalculateLocalSystem(tmp, rRightHandSideVector, rCurrentProcessInfo);
+}
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Public Inquiry
+
+template <class TElementData>
+int SymbolicStokes<TElementData>::Check(const ProcessInfo &rCurrentProcessInfo)
+{
+    KRATOS_TRY;
+    int out = FluidElement<TElementData>::Check(rCurrentProcessInfo);
+    KRATOS_ERROR_IF_NOT(out == 0)
+        << "Error in base class Check for Element " << this->Info() << std::endl
+        << "Error code is " << out << std::endl;
+
+    KRATOS_CHECK_VARIABLE_KEY( DIVERGENCE );
+
+    return 0;
+
+    KRATOS_CATCH("");
+}
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Public I/O
+
+template <class TElementData>
+std::string SymbolicStokes<TElementData>::Info() const
+{
+    std::stringstream buffer;
+    buffer << "SymbolicStokes" << Dim << "D" << NumNodes << "N #" << this->Id();
+    return buffer.str();
+}
+
+template <class TElementData>
+void SymbolicStokes<TElementData>::PrintInfo(
+    std::ostream &rOStream) const
+{
+    rOStream << this->Info() << std::endl;
+
+    if (this->GetConstitutiveLaw() != nullptr){
+        rOStream << "with constitutive law " << std::endl;
+        this->GetConstitutiveLaw()->PrintInfo(rOStream);
+    }
+}
+
+template <class TElementData>
+void SymbolicStokes<TElementData>::Calculate(
+    const Variable<Vector >& rVariable,
+    Vector& rOutput,
+    const ProcessInfo& rCurrentProcessInfo )
+{
+    noalias( rOutput ) = ZeroVector( StrainSize );
+
+    if (rVariable == FLUID_STRESS) {
+
+        // creating a new data container that goes out of scope after the function is left
+        TElementData dataLocal;
+
+        // transferring the velocity (among other variables)
+        dataLocal.Initialize(*this, rCurrentProcessInfo);
+
+        Vector gauss_weights;
+        Matrix shape_functions;
+        ShapeFunctionDerivativesArrayType shape_derivatives;
+
+        // computing DN_DX values for the strain rate
+        this->CalculateGeometryData(gauss_weights, shape_functions, shape_derivatives);
+        const unsigned int number_of_gauss_points = gauss_weights.size();
+
+        double sumOfGaussWeights = 0.0;
+
+        for (unsigned int g = 0; g < number_of_gauss_points; g++){
+
+            UpdateIntegrationPointData(dataLocal, g, gauss_weights[g], row(shape_functions, g), shape_derivatives[g]);
+
+            const Vector gauss_point_contribution = dataLocal.ShearStress;
+
+            noalias( rOutput ) += gauss_point_contribution * gauss_weights[g];
+            sumOfGaussWeights += gauss_weights[g];
+        }
+
+        for (unsigned int i = 0; i < StrainSize; i++){
+            rOutput[i] = ( 1.0 / sumOfGaussWeights ) * rOutput[i];
+        }

```

Comment:
This should be implemented in the base class (not done yet).

Reply:
Now, this is implemented in the base class (in https://github.com/KratosMultiphysics/Kratos/pull/7181)